### ISSUE TITLE
Auxilliary date overwritten by autoformat

### ISSUE THREAD

**Issue by lecagot** [ledger/vim-ledger#36]
While in visual mode and selecting a transition to run the auto-format (here with `gq`), the invoked `ledger` command overwrites the auxiliary date with the first date.

The entry

``` ledger
2016-01-29=2016-01-26 * gin tonic
    expenses:cash   -10.00 EUR
    expenses:out
```

becomes

``` ledger
2016-01-29=2016-01-29 * gin tonic
    expenses:cash                    -10.00 EUR
    expenses:out
```

Any way to change this behaviour or is this a bug?

**Comment by kljohann** [comment]
This looks like a bug to me, as its working for me with `2016/01/29` style dates. Do you use any command line arguments or do you have anything in your `.ledgerrc` file?

**Comment by lecagot** [comment]
yes. my `.ledgerrc` specifies the dateformat:

```
--date-format %Y-%m-%d
```

removing the `--date-format` setting from `.ledgerrc` solves the problem, so it looks like a bug.

**Comment by alerque** [comment]
Is this related to PR #61? If so please see my comments there about whether or not to integrate the fix. I'd be glad to hear some feedback on the subject by whomever is affected.

**Comment by alerque** [comment]
@lecagot As of just now when I merged #61, you can now configure `g:ledger_date_format` to match your preferred format. Note this only applies to `ledger` not `hledger`. I believe this will allow you to fix your issue. If not please feel free to comment back here and we'll re-evaluate.
